drivers/bridge/bridge.go

```diff
@@ -144,7 +144,7 @@ func newDriver() *driver {
 }
 
 // Init registers a new instance of bridge driver
-func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
+func Init(dc driverapi.DriverCallback, ps interface{}, config map[string]interface{}) error {
```

Comment:
PluginGetter naturally falls in DriverRegistry rather than the Controller. I've made necessary updates.